src/List/NonEmpty/Zipper.elm

```diff
@@ -661,21 +661,43 @@ rewindByHelper step n acc =
 -- Functor
 
 
-{-| -}
+{-| Map a function over Zipper
+
+    map (\x -> x + 1) (custom [1] 2 [3, 4])
+    |> toList
+    --> [2, 3, 4, 5]
```

Comment:
I'm not sure why `elm/core` uses functions like this as an example. I think it's important to show that type can be changed within the map while this function simply operates as `(a -> a)`.

Reply:
This was from my head actually :D Added String.fromInt